docs/contributing/standards.rst

```diff
@@ -169,6 +169,10 @@ to indicate that a PR is acceptable. Any code change--other than
 a simple typo fix or one-line documentation change--requires at least two
 maintainers to accept it.
 
+No pull requests can be merged until at least one core maintainer signs off
```

Comment:
I would tend to agree since we don't have any official documentation describing a maintainer's role.